containers/overview/IndexSection.js

```diff
@@ -27,7 +27,8 @@ const IndexSection = ({ pages }) => {
 };
 
 IndexSection.propTypes = {
-    pages: PropTypes.array
+    pages: PropTypes.array,
+    history: PropTypes.object
```

Comment:
I would add `history: PropTypes.object.isRequired` here and in the other places used

Reply:
You're right.